### DISCUSSION TITLE
webOS HyperHDR HDR-Function won't turn on

### DISCUSSION THREAD

**Discussion by kapellmeister84** [awawa-dev/HyperHDR#440] [Q&A]
Hey, I tried the auto LUT calibration with my Mac connected via HDMI, using Chrome browser.
I tried creating a LUT manually. Nothing helps. The HDR button stays off. 
The Problem is, that the colors are way off. Especially green is more cyan.. I dunno what to do.
Help is appreciated!
Thanks :)

**Comment by chesterflaps** [comment]
Have you tried the RGB wizard thing, or calibration wizard?  I've not had a go at messing with LUTs yet but thought I'd offer a possible solution.

**Comment by kapellmeister84** [comment]
Well..yes. The gamma doesn’t need to be corrected at all.. gray looks great. But red is too pink, green and white are too blue. I used Hyperion years ago and somehow it was easier to correct the colors for me, back then..

**Comment by kapellmeister84** [comment]
The funny thing is, when I use the YouTube video for the color calibration, the colors are off. But when I select them in WLED they’re perfect.
